.github/workflows/release.yml

```
@@ -7,7 +7,8 @@ on:
 
 env:
   GHCR_IMAGE: ghcr.io/${{ github.repository }}
-  DOCKERHUB_IMAGE: ${{ github.repository }}
+  DOCKERHUB_IMAGE: docker.io/${{ github.repository }}
+  DOCKERHUB_SIGNATURE_IMAGE: docker.io/${{ github.repository }}-signatures
```

Since we only use this once as `COSIGN_REPOSITORY`, I would rather use _that_ name instead of using both `COSIGN_REPOSITORY` _and_ `DOCKERHUB_SIGNATURE_IMAGE`. Since the name matches the repository with a `-signatures` suffix, this avoids a remapping without losing the meaning we are trying to imply here.

```suggestion
  COSIGN_REPOSITORY: docker.io/${{ github.repository }}-signatures
```

---

Yeah we can. I kept it separate in case we want to add another step to also sign in GHCR.